sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py

```diff
@@ -274,3 +275,21 @@ def test_sale_product_packaging_container_deposit_quantities_case7(self):
         order_line.qty_delivered = 200
         self.assertEqual(pallet_line.qty_delivered, 0)
         self.assertEqual(box_line.qty_delivered, 8)
+
+    def test_sale_product_packaging_container_deposit_quantities_case8(self):
+        """Test add and delete container deposit lines"""
+        sale_form = Form(self.env['sale.order'])
+        sale_form.partner_id = self.sale_order.partner_id
+        with sale_form.order_line.new() as line:
+            line.product_id = self.product_a
+            line.product_uom_qty = 280
+        sale = sale_form.save()
+        with sale_form.order_line.edit(0) as line:
+            line.product_uom_qty = 10
+        sale_form.save()
```

Comment:
That test is falling with` odoo.exceptions.MissingError: Record does not exist or has been deleted.`